src/update_handler.c

```diff
@@ -275,6 +275,7 @@ static gboolean casync_extract_image(RaucImage *image, gchar *dest, GError **err
 
 	res = TRUE;
 out:
+	g_free(seed);
```

Comment:
Good catch! But should be solved in a different way as we've just introduced usage of glib automatic variable cleanup feature. Thus please use instead:
``` diff
--- a/src/update_handler.c
+++ b/src/update_handler.c
@@ -219,7 +219,7 @@ static gboolean casync_extract_image(RaucImage *image, gchar *dest, GError **err
        GError *ierror = NULL;
        gboolean res = FALSE;
        RaucSlot *seedslot = NULL;
-       gchar *seed = NULL;
+       g_autofree gchar *seed = NULL;
        gchar *store = NULL;
        gboolean seed_mounted = FALSE;
```